package.json

```diff
@@ -38,10 +40,13 @@
     "handlebars": "^3.0.0",
     "hapi": "^8.2.0",
     "hapi-auth-cookie": "^3.0.1",
+    "highlight.js": "^8.6.0",
     "joi": "^5.1.0",
     "lodash": "^3.3.1",
+    "marked": "^0.3.3",
     "mysql": "^2.5.5",
     "prompt": "^0.2.14",
+    "ua-parser-js": "^0.7.9",
```

Comment:
I overlooked `platform.src.js` on the client. I was working backwards from [test page](https://github.com/maxbeatty/jsperf.com/pull/1/files#diff-3089e3bc030ce0d2deec583de5ec8723L97) to [`userAgent`](https://github.com/maxbeatty/jsperf.com/pull/1/files#diff-6a2d247a6d06eee315e37e7696833f79L94) to [browser.php](https://github.com/maxbeatty/jsperf.com/blob/master/_inc/browser.php) so just tried to find a suitable replacement. Looks like I can remove user agent on the server altogether